docs/reference/settings/audit-settings.asciidoc

```diff
@@ -40,29 +48,39 @@ by using the following settings:
 // tag::xpack-sa-lf-events-include-tag[]
 `xpack.security.audit.logfile.events.include`::
 (<<dynamic-cluster-setting,Dynamic>>)
-Specifies which events to include in the auditing output. The default value is:
+Specifies the kind of events to print in the auditing output. The default list value
+contains:
 `access_denied, access_granted, anonymous_access_denied, authentication_failed,
-connection_denied, tampered_request, run_as_denied, run_as_granted`.
+connection_denied, tampered_request, run_as_denied, run_as_granted,
+security_config_change`.
 // end::xpack-sa-lf-events-include-tag[]
 
 [[xpack-sa-lf-events-exclude]]
 // tag::xpack-sa-lf-events-exclude-tag[]
 `xpack.security.audit.logfile.events.exclude`::
 (<<dynamic-cluster-setting,Dynamic>>)
-Excludes the specified events from the output. By default, no events are
-excluded.
+Excludes the specified kind of events from the include list. Useful in the case where
+the `events.include` setting contains the special value `_all`. The default is the empty list.
 // end::xpack-sa-lf-events-exclude-tag[]
 
 [[xpack-sa-lf-events-emit-request]]
 // tag::xpack-sa-lf-events-emit-request-tag[]
 `xpack.security.audit.logfile.events.emit_request_body`::
 (<<dynamic-cluster-setting,Dynamic>>)
-Specifies whether to include the request body from REST requests on certain
-event types such as `authentication_failed`. The default value is `false`.
+Specifies whether to include the full request body from REST requests as an
+attribute of certain kinds of audit events. The request body is printed as an
+escaped JSON string value to the `request.body` attribute.
+These events that contain the `request.body` attribute are `authentication_success`,
+`authentication_failed`, `realm_authentication_failed`, `tampered_request`, `run_as_denied`,
+and `anonymous_access_denied` that are printed on the coordinating node only
```

Comment:
> that are printed on the coordinating node only...

Does this phrase apply to all the items in the list, or only the last one?  If it applies to all of them, it might be clearer to put it in a separate sentence. For example: "...and `anonymous_access_denied`. This information is printed in the audit log on the coordinating node only..."

Reply:
You're right, it's confusing to talk about that in this context.
In a75acb753215bd41030 I've moved these clarifications to the dedicated page https://elasticsearch_66839.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/auditing-search-queries.html and linked to it here.